api/OffscreenCanvas.json

```
@@ -409,14 +409,14 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "preview"
+                "version_added": "17.4"
```

@jdatapple Can you confirm whether `webgpu_context` is intentionally exposed in worker contexts?

---

@queengooborg As long as we have on clarity, I would suggest to play safe, revert this bit, and stash this outstanding question to an issue.

```suggestion
                "version_added": "preview"
```